subprojects/scala/src/main/java/org/gradle/api/tasks/scala/ScalaDoc.java

```diff
@@ -198,6 +210,7 @@ protected void generate() {
             parameters.getClasspath().from(getClasspath());
             parameters.getOutputDirectory().set(getDestinationDir());
             parameters.getSources().from(getSource());
+            parameters.getIsScala3().set(getIsScala3());
```

Comment:
As indicated, reach out to the `ScalaBasePlugin` instead

Reply:
Removed